src/test/java/com/bieganski/ox/BoardTest.java

```diff
@@ -0,0 +1,25 @@
+package com.bieganski.ox;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class BoardTest {
+    @Test()
```

Comment:
No need for parens.
No need for the annotation actually, since there's one on class.
Magic numbers in the test - shall it only works for Board(10), Coords(5,5) and when X is inputted? And if the board is larger or smaller, or char is an O?
Before return: assert on message being as you expect it.